docs/website/data/integrations.yaml

```diff
@@ -1851,3 +1864,6 @@ software:
   graphql:
     name: graphql
     link: https://graphql.org/
+  ccbr:
+    name: ccbr
+    link: https://github.com/niuzhi/ccbr
```

Comment:
I'm a little confused... Is it a commercial offering? How would I find out more about it? The github repo seems to host just the screenshots 👀

Reply:
@srenatus 
Hello, the code is currently undergoing an internal compliance scan and is expected to be released to the github repository next week. I will contact you for review when the code is merged. thanks

Reply:
Awesome. Let's close this for now, and reopen when there's some code. OK with you? Right now, it would feel a bit weird to have this empty box on the shelf 😅

Reply:
ok，thanks